prime-router/src/main/kotlin/azure/observability/event/AzureEventUtils.kt

```diff
@@ -34,4 +38,25 @@ object AzureEventUtils {
     fun getIdentifier(bundle: Bundle): MessageID {
         return MessageID(bundle.identifier?.value, bundle.identifier?.system)
     }
+
+    /**
+     * Returns a ReportSentEvent
+     */
+    fun createReportSentEvent(
+        receiver: Receiver,
+        reportFiles: List<ReportFile?>,
+        reportId: ReportId,
+        externalFilename: String,
+    ): ReportSentEvent {
+        val reportFile = if (reportFiles.size == 1) reportFiles.first() else null
```

Comment:
I'm not sure this right? For a batch of items wouldn't it be possible for there be more than one root report?